src/hooks/useProjectVariablesUpdate/util.ts

```diff
@@ -184,7 +184,7 @@ export const makeVariableValueOperations = ({
           value: nV,
           existingValueId: pendingValues[index].id,
         });
-      } else {
+      } else if (nV.textValue) {
```

Comment:
If `nV.textValue` isn't set here, there's no operation to perform because no text is being added or updated.